internal/wasm/interpreter/interpreter.go

```diff
@@ -161,22 +140,25 @@ type callFrame struct {
 	// pc is the program counter representing the current position in compiledFunction.body.
 	pc uint64
 	// f is the compiled function used in this function frame.
-	f *compiledFunction
+	f *compiledFunctionInstance
 }
 
 type compiledFunction struct {
-	moduleEngine *moduleEngine
-	source       *wasm.FunctionInstance
-	body         []*interpreterOp
-	hostFn       *reflect.Value
+	body   []*interpreterOp
+	hostFn *reflect.Value
+}
+
+type compiledFunctionInstance struct {
```

Comment:
It isn't obvious why there are two types here, can you add a comment to each (also in JIT)?

note: least favorite naming suffix Is instance. I see why you are doing it because we have this Instance suffix in other types due to literal translation of spec names. It is like saying I have a string instance.

Reply:
so renamed to `compiledFunction` and `compiledFunctionInstance` to simply `code` and `function`.